docs/conf.py

```diff
@@ -0,0 +1,39 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Add the project root to the Python path ---------------------------------
+# This is necessary to discover all the spras source code, otherwise auto-population of
+# doc strings will be scoped to the `docs/` directory.
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('..'))
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'SPRAS'
+copyright = '2025, Anthony Gitter & Anna Ritz'
+author = 'Anthony Gitter & Anna Ritz'
+release = '0.2.0'
```

Comment:
Can we pull this version from somewhere else? I have now bumped to `0.3.0` and will soon bump again to test the Docker action.

Reply:
One related project https://github.com/search?q=repo%3Amanubot%2Fmanubot%20__version__&type=code

Reply:
I did some poking around, and it looks like the way `manubot` does has [fallen slightly out of flavor](https://stackoverflow.com/questions/72167802/adding-version-attribute-to-python-module). Instead, I'll use the approach suggested in the top SO answer, where I set it in `pyproject.toml` and use that to populate other places (including setting the canonical `__version__` dunder).

Reply:
We discussed your new solution, and I like it. We may need a try/except in case the package is not installed. This also makes stronger assumptions that spras runs as an installed package, so we plan to update the readme and test case accordingly so the test case installs the local source into the activated conda environment.